devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy

```diff
@@ -112,7 +112,7 @@ metadata {
 
 
 		main(["status", "acceleration", "temperature"])
-		details(["status", "acceleration", "temperature", "3axis", "battery", "refresh"])
```

Comment:
@dkirker , correct, it will still be able to generate events without the valueTile.